stratum/hal/lib/barefoot/bfrt_node.cc

```diff
@@ -278,15 +192,23 @@ ::util::Status BfrtNode::CommitForwardingPipelineConfig() {
   return ::util::OkStatus();
 }
 
-::util::Status BfrtNode::VerifyForwardingPipelineConfig(
-    const ::p4::v1::ForwardingPipelineConfig& config) const {
+::util::Status BfrtNode::DoVerifyForwardingPipelineConfig(
```

Comment:
The `DoVerifyX` vs `VerifyX` in the bcm code tree exists to ensure correct locking of the chassis. I'm also not convinced that a verify method should extract something.

Reply:
I made an additional commit on top of these and could even remove one `extract` call: https://github.com/stratum/stratum/pull/366